plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzers/base_analyzer.py

```diff
@@ -156,7 +158,7 @@ def full_analysis(self):
                 except:  # pylint: disable=bare-except
                     pass
 
-        return self._images_created if len(self._images_created) > 0 else []
+        return self._images_created if len(self._images_created) > 0 else [], [], []
```

Comment:
This change means that now, all class who would like to add a data frame or a text, will need to overwride this method.

We should have done this change in a way the full analysis of the base returns all the images, data frame and texts.

If the analysis function of the images, handles adding the images to the images list, we can do the same for data frames and the texts.
This way, we won't need overrides in the analyzers, "just" to make sure that each analysis function adds the data to the right location in the class

Reply:
I made changes to the function; it no longer returns anything. Instead, I added getters to retrieve images, data frames, or text separately.